code/game/objects/items/storage/pouch.dm

```diff
@@ -532,6 +532,46 @@
 	new /obj/item/storage/pill_bottle/tramadol(src)
 	new /obj/item/stack/medical/splint(src)
 
+/obj/item/storage/pouch/medkit/marine
+	name = "medkit pouch"
+	desc = "A standard use medkit pouch that can contain all kinds of stuff. This marine version has 6 slots."
+	icon_state = "medkit"
+	w_class = WEIGHT_CLASS_BULKY //does not fit in backpack
+	max_w_class = 4
+	storage_slots = 6
+	can_hold = list(
+		/obj/item/healthanalyzer,
+		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/pill,
+		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/storage/pill_bottle,
+		/obj/item/stack/medical,
+		/obj/item/flashlight/pen,
+		/obj/item/storage/pill_bottle/packet,
+		/obj/item/reagent_containers/hypospray,
+	)
+
+/obj/item/storage/pouch/medkit/corpsman
```

Comment:
this is just the normal medkit, you changed nothing compared to the parent.

Either you make /obj/item/storage/pouch/medkit the nerfed version, or you make /obj/item/storage/pouch/medkit/marine but then you have to search in the whole code for /obj/item/storage/pouch/medkit and replace it everywhere to prevent pfc from having access to it